### PR TITLE
Create a new UDF file for Tutorial

### PR DESCRIPTION
**Pull request by dsk-2015**
## Purpose

To incorporate Cameron & Brian's feedback that it's easy to fat-finger when you add snippets to an existing file. A new file will act as a reference.

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] New sample or new feature within a sample
[ ] Code style update (formatting, naming)
[X ] Refactoring (no functional changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## Test By
dotnet run GetAvailableAndFreshSpaces

Please describe how you tested

Note: I got null values for the sensors a couple times. The rest of the times, it worked fine.

### CODE REVIEW

File: occupancy-quickstart/src/actions/userDefinedFunctions/availabilityForTutorial.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+       
+       // Retrieve carbonDioxide, temperature, and motion sensors
+       var carbonDioxideSensor = otherSensors.find(function(element) {
+           return element.DataType === carbonDioxideType;
+       });
+        var motionSensor = otherSensors.find(function(element) {
+           return element.DataType === motionType;
+       });
+       var temperatureSensor = otherSensors.find(function(element) {
+       return element.DataType === temperatureType;
```
Comment (fernandoviton) [inline occupancy-quickstart/src/actions/userDefinedFunctions/availabilityForTutorial.js]: indentation is off

File: occupancy-quickstart/src/actions/userDefinedFunctions/availabilityForTutorial.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+       if(carbonDioxideValue === null || motionValue === null || temperatureValue === null){
+           sendNotification(telemetry.SensorId, "Sensor", "Error: Carbon dioxide, motion, or temperature are null, returning");
+           return;
+       }
+        
+       var alert = "Room with fresh air and comfortable temperature is available.";
+       var noAlert = "Either room is occupied, or working conditions are not right.";
+    
+       // If sensor values are within range and room is available
+       if(carbonDioxideValue < carbonDioxideThreshold && temperatureValue < temperatureThreshold && presence) {
```
Comment (fernandoviton) [inline occupancy-quickstart/src/actions/userDefinedFunctions/availabilityForTutorial.js]: Shouldn't this be !presence or am I missing something?  If so it is also wrong in availability.js.

Comment (fernandoviton) [reply occupancy-quickstart/src/actions/userDefinedFunctions/availabilityForTutorial.js]: Oh I messed up.  It is right in availability.js - I think you just need to update here.

Comment (dsk-2015) [reply occupancy-quickstart/src/actions/userDefinedFunctions/availabilityForTutorial.js]: You are right! Thanks! I'll fix and test this.

**Comment by dsk-2015** [comment]
Testing seems fine. Updated with the 2 fixes you pointed out. Merging now.

**Pull request merged**